src/vue/components/word-usage-examples/word-usage-examples-sorting.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div v-if="availableSortBy" class="alpheios-word-usage-header-sorting">
+    <div v-show="availableSortBy" class="alpheios-word-usage-header-sorting">
```

Comment:
I have changed to `v-show` because this block is always the same - we need only to change visibility here and don't need re-rendering